pkgs/development/python-modules/pycairo/default.nix

```diff
@@ -23,13 +23,16 @@ if (isPyPy || isPy35) then throw "pycairo not supported for interpreter ${python
     sha256 = "0xfl1i9dips2nykyg91f5h5r3xpk2hp1js1gq5z0hwjr0in55id4";
   };
 
+  patch_waf-py3_5 = ./waf-py3_5.patch;
```

Comment:
You can also do `patches = [ ./waf-py3_5.patch ]`; then the patch is applied automatically by a phase hook, but I can see that it makes sense to do it this way, as well, in keeping with how the other patch is applied (I'd move both to `patches`)